src/vs/platform/extensionManagement/common/extensionManagement.ts

```diff
@@ -93,6 +93,11 @@ export interface IColor {
 	defaults: { light: string, dark: string, highContrast: string };
 }
 
+export interface IExtensionsConfigContent {
+	recommendations: string[];
+	unwantedRecommendations: string[];
+}
+
```

Comment:
Lets move this right after `IIgnoredRecommendations ` such that all recommendations/tips related interfaces are together and helps readability